---
layout: post
title:  "J. Christopher Bare"
category: resume
---

- **Email**: {middlename}{lastname} at gmail
- **Location**: Wellington, New Zealand / Seattle, WA
- **LinkedIn**: [linkedin.com/in/j-christopher-bare](https://linkedin.com/in/j-christopher-bare)  
- **GitHub**: [github.com/cbare](https://github.com/cbare)


## Skills

Software engineer focused on data and machine learning.

- **Skills**: Python, SQL, Snowflake, AWS, Azure, Git, Bash, Docker, APIs, machine learning, data pipelines, CI/CD, automated testing, code review.
- **Other experience**: Keras, Tensorflow, scikit-learn, R, JavaScript, React, Redux, Java, Azure, Spark, Scala, Haskell, Clojure, Scheme, 6502 assembler.


## Professional Experience

### Data Engineer / Senior Data Engineer  
**Volpara Health** – Wellington, New Zealand  
*Mar. 2021 - Present*

- Built data platform for de-identified clinical data. Python, DICOM, Azure, Snowflake.
- Curated image datasets and annotations used to develop computer vision models for mammography.
- Prototyped image segmentation model to detect arterial calcification. PyTorch.

### Senior Software Engineer  
**98point6** – Seattle, WA  
*Sept. 2019 - Aug. 2020*

- Applied machine learning to on-demand text-based primary care medicine.
- MLOps for an NLP service for automated patient interview. Python, Flask, Keras, TensorFlow, scikit-learn, fastText, PostgreSQL, AWS.

### Senior Software Engineer  
**Arivale** – Seattle, WA  
*Feb. 2017 - May 2019*

- Contributed to a [health and wellness software platform](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5568837/) integrating data from genetics, blood, microbiome, diet, and activity. Python, PostgreSQL, AWS, React.
- Worked with clinicians and researchers on a machine learning pipeline to make health recommendations based on lifestyle, genetics, and blood analytes. Python, pandas, scikit-learn, AWS.
- Maintained a pipeline that delivered a daily snapshot of cleaned, normalized, and batch-corrected data to researchers. R, data.table, Python, pandas, numpy, PostgreSQL, AWS.
- Built end-to-end system to collect medication and supplement usage classified by indication and mode of action. Python, R, PostgreSQL, AWS, React.

### Senior Software Engineer  
**Sage Bionetworks** – Seattle, WA  
*Aug. 2012 - Aug. 2016*

- Developed client libraries for [Synapse](https://www.synapse.org/), a data sharing platform for collaborative scientific data analysis exposing a [ReST API](http://python-docs.synapse.org/) in Python and R.
- Developed [code](https://github.com/cbare/SynapseChallengeTemplates) to manage predictive modeling and machine learning competitions on biomedical research questions as part of [DREAM challenges](http://dreamchallenges.org/). Python, R.
- Helped scientists apply version control, cloud computing, and Synapse to open and reproducible research.

### Software Engineer  
**Institute for Systems Biology** – Seattle, WA  
*Jun. 2006 - Jul. 2012*

- Developed [Network Portal](http://networks.systemsbiology.net/), a database of gene regulatory networks. Python, Django, PostgreSQL, Solr, Javascript, jQuery, R.
- Created an [interactive visualization of gene expression](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-382). Java, Swing, SQLite, R.
- Implemented [Firegoose](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-456), a Firefox browser extension for data exchange between desktop and web applications. Javascript, Java, Web services, Microformats.

### Prior experience

- Software development for applications in genomics, social media, e-commerce, pharmaceuticals, and biomedical research.</a> Java, Swing, SQL, JavaScript.

## Education

### Continuing  
Machine learning, deep learning, natural language processing, data engineering, and functional programming in Scala and Haskell.

### M.S., Computer Science & Engineering  
**University of Washington** – Seattle, WA  
*Jun. 2004 - Jun. 2007*

### B.S., Mathematical Science, Minor in Computer Science  
**Pennsylvania State University** – University Park, PA  
*Jun. 1989 - Aug. 1992*


## Selected Publications

- [**The mPower study, Parkinson disease mobile data collected using ResearchKit**](http://www.nature.com/articles/sdata201611). Bot et al., *Nature Scientific Data*, 2016.
- [**Prediction of human population responses to toxic compounds by a collaborative competition**](https://www.nature.com/articles/nbt.3299). Eduati et al., *Nature Biotechnology*, 2015.
- [**Simulation Studies as Designed Experiments: The Comparison of Penalized Regression Models in the “Large p, Small n” Setting**](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0107957). Neto et al., *PLOS One*, 2014.
- [**Global optimization of somatic variant identification in cancer genomes with a global community challenge**](http://www.nature.com/ng/journal/v46/n4/full/ng.2932.html). Boutros et al., *Nature Genetics*, 2014.
- [**Architecture for interoperable software in biology**](http://bib.oxfordjournals.org/content/15/4/626)
  Bare et al., *Briefings in bioinformatics*, 2012
- [**Parallel evolution of transcriptome architecture during genome reorganization**](https://genome.cshlp.org/content/21/11/1892)
  Yoon et al., *Genome Research*, 2011
- [**Integration and visualization of systems biology data in context of the genome**](http://www.biomedcentral.com/1471-2105/11/382/) Bare et al., *BMC Bioinformatics*, 2010
- [**Prevalence of transcription promoters within archaeal operons and coding sequences**](https://www.embopress.org/doi/full/10.1038/msb.2009.42) Koide et al., *Molecular Systems Biology*, 2009
- [**The Firegoose: two-way integration of diverse data from different bioinformatics web resources with desktop applications**](http://www.biomedcentral.com/1471-2105/8/456) Bare et al., *BMC Bioinformatics*, 2007


[More publications](https://scholar.google.com/citations?user=EEZ5DLgAAAAJ&hl=en).


*Last updated: October 2024.*
